Example/Podfile

```diff
@@ -1,5 +1,7 @@
 use_frameworks!
 
+platform :ios, 8.0
```

Comment:
We may want to change the minimum to iOS 9 at some point, but 8 should still work for now.